README.md

```diff
@@ -16,3 +16,12 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $f(n)\in o(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$
+
+$f(n)\in o(g(n)) \implies  f(n)\in O(g(n))$
+  
+$\forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)  \implies \forall c>0, \exists n_0, \forall n\ge n_0: f(n) \le c g(n)$
+  
+Choose $n_0$ such that $O(g(n)) \subset o(g(n))$
```

Comment:
Not following you here -- the $n_0$ is the same in both definitions. What are the differences?